src/runtime_src/core/pcie/emulation/common_em/unix_socket.h

```diff
@@ -33,7 +33,7 @@ class unix_socket {
     int fd;
     std::string name;
 public:
-    bool server_started;
+    std::atomic_bool server_started;
```

Comment:
What is wrong with std::atomic<bool> and regular assignment rather than store/load?

Reply:
First of all, load/store member functions usage tells that the variable is not a normal bool but an atomic variable. 
Second, it provides the platform to change default memory ordering while executing. In future, that can be easily updated by looking at these function calls.
Note: we are using sequentially consistent memory ordering.

Reply:
I agree with the communication of intent with store() and load().